### ISSUE TITLE
Missing --submit for the add/remove tags operations

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#276]
Originally reported of Google Code with ID 220.

```
for kind in instance node ; do for op in add remove; do gnt-$kind $op-tags --help; done; done|grep submit

<empty output>
```
Originally added on 2012-03-02 09:21:03 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
gnt-instance migrate has no "--submit" too, and many other non-interactive commands of gnt-instance lack this option.
```
Originally added on 2012-03-02 10:09:01 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-05-09 05:56:37 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
I'll take this (as it's partially needed for adding the -C option to hroller).
```
Originally added on 2013-05-10 11:51:24 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Actually, this bug should have been closed long ago.

For gnt-node it was fixed in c5a66db by Michael Hanselmann
For gnt-instance it was fixed in f70bb62 by Michael Hanselmann

Both changes made it into the 2.6.0 release.
```
Originally added on 2013-05-10 12:25:29 +0000 UTC.

Changed State: Released
